hackathon_contributions_new_data_format/create_petab_yaml.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import os
+import pandas as pd
+import yaml
+import petab
+
+from copy import deepcopy
+
+
+def create_petab_yaml(model_dir):
+    filename = f'{model_dir}.yaml'
+    config = {
+        'petab_version': petab.__version__,
+        'parameter_file':
+            petab.get_default_parameter_file_name(model_dir),
+        'problems': [
+            {
+                'sbml_file':
+                    petab.get_default_sbml_file_name(model_dir),
+                'condition_file':
+                    petab.get_default_condition_file_name(model_dir),
+                'measurement_files':
+                    [
+                        petab.get_default_measurement_file_name(model_dir),
+                    ],
+            },
+        ]
+    }
+
+    data = [
+        (filename, config),
+    ]
+
+    if model_dir == 'Becker_Science2010':
+        config['problems'].append(deepcopy(config['problems'][0]))
+        config['problems'][0]['sbml_file'] = \
+            'model_Becker_Science2010__BaF3_Exp.xml'
+        config['problems'][0]['condition_file'] = \
+            'experimentalCondition_Becker_Science2010__BaF3_Exp.tsv'
+        config['problems'][0]['measurement_files'][0] = \
+            'measurementData_Becker_Science2010__BaF3_Exp.tsv'
+        config['problems'][1]['sbml_file'] = \
+            'model_Becker_Science2010__binding.xml'
+        config['problems'][1]['condition_file'] = \
+            'experimentalCondition_Becker_Science2010__binding.tsv'
+        config['problems'][1]['measurement_files'][0] = \
+            'measurementData_Becker_Science2010__binding.tsv'
+        # simulationData_Becker_Science2010__BaF3_Exp.tsv
+        # simulationData_Becker_Science2010__binding.tsv
+
+    elif model_dir in ('Casaletto_PNAS2019', 'Merkle_PCB2016'):
```

Comment:
Why do the yaml files for Casaletto and Merkle exist? Shouldn't this prevent them from being created?

Reply:
That was from an earlier version not caring about missing files. Will remove.